src/components/Table/Budget/BudgetLineItemTable/__test__/BudgetLineitemTable.test.tsx

```diff
@@ -158,7 +158,7 @@ beforeEach(() => {
 
 describe("Budget Line Item Table tests", () => {
 	test("renders correctly", async () => {
-		// await waitForElement(() => table.getByText(/500/i));
-		// await waitForElement(() => table.getByText(/note 1/i));
+		await waitForElement(() => table.getByText(/500/i));
```

Comment:
what is /500/i?

Reply:
This data is mentioned in the mock file and I am checking if it is rendered in the table.